src/database/database.cpp

```diff
@@ -64,7 +65,7 @@ const std::unordered_map<std::string, std::string> Database::kVendorNames{
 // Constructor, computing device properties and populating the parameter-vector from the database.
 // This takes an optional overlay database in case of custom tuning or custom kernels.
```

Comment:
Minor note: the comment is still referring to the old overlay database, not to the new plugin system